src/main/java/Server/MopeLSPServerLauncher.java

```diff
@@ -94,11 +108,14 @@ public static void readConfigFile(Path path) throws IOException {
             prop.load(bufferedReader);
             configObject.port = Integer.parseInt(prop.getProperty("server.port"));
             configObject.path = prop.getProperty("server.path");
-            logger.info("Read Port " + configObject.port + " from " + path);
+            logger.debug("Read Port " + configObject.port + " from " + configObject.path);
             bufferedReader.close();
         }
     }
 
+    /**
+     * <p>Trying to read a configfile from the standard config directory according to the OS is using</p>
```

Comment:
You definitely have to go into detail here. Which paths are tried in which OS? Which options take precedence over others? What happens if the config is not found in any of the paths? Which Exception is thrown if there is a corrupted config file? ...